src/ai_client.c

```diff
@@ -3,6 +3,8 @@
  *
  * Copyright (C) 2009-2011 Keldon Jones
  *
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
As it is now, we cannot connect to the official server. We must fake our information and send "0.9.4" instead of VERSION on line 2349.